### DISCUSSION TITLE
Is it normal for the decection service to return the same age(25-32)?

### DISCUSSION THREAD

**Discussion by IldarGreat** [exadel-inc/CompreFace#896] [Q&A] [answered]
I just started used the ComperFace and noticed that no matter what picture I upload, the return of information about the person age will be the same: low(25) and hight(32). Is this normal or did I miss something when I ran it?
![Снимок экрана (329)](https://user-images.githubusercontent.com/90307025/190473632-05cab9f7-27e8-4d92-a06d-bfdb15472a56.png)

**Comment by pospielov** [comment, marked as answer]
![image](https://user-images.githubusercontent.com/3736126/191317465-46432d6b-d586-4d4e-9d43-a638e89ae36b.png)
It returns different results, but it's not very accurate.
I would recommend using [InsightFace custom build](https://github.com/exadel-inc/CompreFace/blob/master/docs/Custom-builds.md), it's more accurate then default
